example/lib/pages/map/tiled/tiled_page.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:bonfire/bonfire.dart';
+import 'package:example/shared/decoration/spikes.dart';
+import 'package:flutter/widgets.dart';
+
+class TiledPage extends StatelessWidget {
+  const TiledPage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    const tileSize = 16.0;
+    return BonfireWidget(
+      joystick: Joystick(directional: JoystickDirectional()),
```

Comment:
Running on the web, the speed it moves the camera is very fast, can we slow it down a bit?

Reply:
Also, sometimes the tiles have a little displacement, the border lines get more thick after stopping moving the camera.

Reply:
This must be a rounding problem with Flutter's rendering system. I don't think I have much control over that.

Reply:
When you apply more zoom, this behaviour not show.

Reply:
If you have any idea that how resolve this problem. Help me. rsrsrs

Reply:
I see, I can't think of anything to solve this either :(
But it seems better now after the changes